pkg/exporter/consul_exporter.go

```diff
@@ -76,6 +76,11 @@ var (
 		"Tags of a service.",
 		[]string{"service_id", "node", "tag"}, nil,
 	)
+	serviceMeta = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_meta"),
```

Comment:
This seems like an "info" metric, should end in `_info`.

```suggestion
		prometheus.BuildFQName(namespace, "", "service_meta_info"),
```

Reply:
Changing this would be inconsistent with the other things we're exporting. Probably the most pertinent example is `service_tag`. I can't see any other `info` metrics in the codebase.

Reply:
Those can be fixed later. I would prefer to have new metrics follow [best practices](https://prometheus.io/docs/practices/naming/#metric-names)